cmd/flow-rosetta-server/main.go

```diff
@@ -126,14 +126,14 @@ func run() int {
 
 	// Initialize the SDK client.
 
-	if flagFlowAPI == "" {
-		log.Error().Msg("Flow API endpoint is missing")
+	if flagAccessAPI == "" {
+		log.Error().Msg("Flow Access API endpoint is missing")
 		return failure
 	}
 
-	flowClient, err := client.New(flagFlowAPI, grpc.WithInsecure())
+	flowClient, err := client.New(flagAccessAPI, grpc.WithInsecure())
```

Comment:
I think we should call the clients `accessAPI` and `dpsAPI` respectively. Or, alternatively, `accessClient` and `dpsClient` at least. Though the former imo makes more sense.

Reply:
I did not follow the conversation on this. In the end do we depend on the real Flow Access API or our own here?

Reply:
This one is for the Flow Access API, in order to submit the transaction for execution.